include/os/freebsd/spl/sys/vnode.h

```diff
@@ -83,6 +85,22 @@ vn_is_readonly(vnode_t *vp)
 #define	vn_has_cached_data(vp)	\
 	((vp)->v_object != NULL && \
 	(vp)->v_object->resident_page_count > 0)
+
+static __inline void
+vn_flush_cached_data(vnode_t *vp, boolean_t sync)
+{
+#if __FreeBSD_version >= 1300125
```

Comment:
`vm_object_mightbedirty` should be available `> 1300054`, but this is close enough. I'm a little curious where the number came from though.

Reply:
Good question, I thought I used the version number from `sys/sys/param.h` (that is the correct place, right?).  But looking at it again today it's clearly `1300054` as you pointed out.  I've made the change.  I must have accidentally grabbed the wrong version from a different branch while looking for this.